powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/service/CredentialCounterService.java

```diff
@@ -137,14 +137,26 @@ public void updateCredentialCounter(CredentialEntity credential, AuthenticationR
     @Transactional
     public ResetCountersResponse resetCounters(ResetCountersRequest request) {
```

Comment:
The request should contain info about the type of reset. Otherwise, the side-effect can be unexpected. For example:

```json
{
    "active": true,
    "blocked": true
}
```

Reply:
I prefer an enumeration parameter because I think there are two use cases here, not four (from the booleans in example): `RESET_BLOCKED`, `RESET_ACTIVE_AND_BLOCKED`.